src/components/notify/index.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import './alert.scss';
+import { Alert } from 'react-bootstrap';
+
+const Notification = (props) => {
```

Comment:
@chingsley is this the component for the alert that shows the success/error message? If yes, I think `Notification` is an ambiguous class name and might conflict with the notification work to be done on the app.